payone-adapter/src/main/java/com/commercetools/sunrise/payment/payone/PayonePaymentServiceProviderImpl.java

```diff
@@ -0,0 +1,51 @@
+package com.commercetools.sunrise.payment.payone;
+
+import com.commercetools.sunrise.payment.domain.CreatePaymentDataProvider;
+import com.commercetools.sunrise.payment.domain.PaymentServiceProvider;
+import com.commercetools.sunrise.payment.utils.PaymentPropertiesLoadingHelper;
+import io.sphere.sdk.payments.Payment;
+import io.sphere.sdk.payments.PaymentMethodInfo;
+import io.sphere.sdk.payments.PaymentStatus;
+
+import java.util.List;
+import java.util.Map;
+import java.util.function.BiFunction;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+
+/**
+ * Created by mgatz on 7/18/16.
+ */
+public class PayonePaymentServiceProviderImpl implements PaymentServiceProvider {
```

Comment:
With Sunrise modules we usually skip the "Impl" suffix if the name is already concrete. So if the implementation would be called the same, then `PaymentServiceProviderImpl` would be necessary. But in this case the `Payone` prefix already indicates it's the implementation of the PSP. So I would just call it `PayonePaymentServiceProvider`.